Sources/SwiftSyntax/Raw/RawSyntaxNodeProtocol.swift

```diff
@@ -148,8 +157,13 @@ public struct RawTokenSyntax: RawSyntaxNodeProtocol {
   /// Creates a missing `TokenSyntax` with the specified kind.
   public init(missing kind: RawTokenKind, arena: __shared SyntaxArena) {
     self.init(
-      kind: kind, text: "", leadingTriviaPieces: [], trailingTriviaPieces: [],
-      arena: arena)
+      kind: kind,
+      text: "",
```

Comment:
Inconsistent with `RawSyntax.makeMissingToken()`. Should we use `kind.defaultText` here?

Reply:
I did actually change that in some follow-up PR but it looks like it didn’t make it into this PR. Updated it now.